README.md

```diff
@@ -1,5 +1,5 @@
 1. Replace `<your_account>` with your Github username in the link
-    - [DEMO LINK](https://<your_account>.github.io/js_task_parse_list_DOM/)
+    - [DEMO LINK](https://rekverr.github.io/js_task_parse_list_DOM/)
```

Comment:
Add this demo link to the PR description 
<img width="943" alt="image" src="https://github.com/user-attachments/assets/0f750f9e-cadf-45c9-b47e-8e49720539ea" />